misc/h2olog/.gitignore

```diff
@@ -0,0 +1,11 @@
+*.swp
+*.old
+*~
+.DS_Store
+CMakeCache.txt
+CMakeFiles/
+Makefile
+cmake_install.cmake
+h2olog
+build/
+.pip3/
```

Comment:
I believe that the contents of this file can be merged to `.gitignore` in the root file?